libs/utils/decorate.js

```diff
@@ -57,7 +70,7 @@ export function decorateBlockText(el, config = ['m', 's', 'm'], type = null) {
       headings.forEach((h) => h.classList.add(`heading-${config[0]}`));
       if (config[2]) {
         headings[0]?.previousElementSibling?.classList.add(`detail-${config[2]}`);
-        decorateIconArea(el);
+        decorateIconArea(el, headings);
```

Comment:
The only concern is this line, before the function not deal with the heading, but it does now, I mean even they have heading there, but not decorate it, after the code change, it decorate heading, I am not sure this change will impact existing content they do have headings, please test and double check

Reply:
@JackySun9 Thanks for the eyes on this and valid concern. 
I have reverted this piece and moved that functionality to the 'text' block, the only place it's used. I did some regression testing on that block and don't see any issues.